backend/geonature/tests/test_validation.py

```diff
@@ -99,3 +129,48 @@
         assert response.status_code == 200
         assert len(response.data) > 0
         assert response.json[0]["id_status"] == str(id_nomenclature_valid_status.id_nomenclature)
+
+    def test_auto_validation(
+        self,
+        users,
+        app,
+        auto_validation_enabled,
+        validation_with_max_score_and_wait_validation_status,
+    ):
+        fct_auto_validation_name = app.config["VALIDATION"][
+            "AUTO_VALIDATION_SQL_FUNCTION"
+        ]  # config["VALIDATION"]["AUTO_VALIDATION_SQL_FUNCTION"]
+        set_logged_user(self.client, users["user"])
+
+        id_nomenclature_probable = db.session.execute(
+            """ select tn.id_nomenclature from ref_nomenclatures.t_nomenclatures tn where tn.mnemonique = 'Probable' """
+        ).scalar()
+        id_nomenclature_attente_validation = db.session.execute(
+            """ select tn.id_nomenclature from ref_nomenclatures.t_nomenclatures tn where tn.mnemonique = 'En attente de validation' """
+        ).scalar()
+
+        list_synthese_to_update = []
+        for row in validation_with_max_score_and_wait_validation_status:
+            list_synthese_to_update.append(row[2])
+
```

Comment:
**Suggestion:** Consider using a list comprehension to simplify your code and improve readability. [enhancement]
```suggestion
        list_synthese_to_update = [row[2] for row in validation_with_max_score_and_wait_validation_status]
```